FAQs/README.md

```diff
@@ -114,6 +116,25 @@ Using the native enterprise directory for authentication ensures that there is a
 <br>
 <br>
 
+### **7. How can I migrate classic Application Insights to Workspace-based? If I migrate Application Insights AzSK-AzTS-AppInsights-xxxxx to Workspace-based, will there be any impact to AzTS scan?**
+It is recommended to migrate to classic Application Insights to Workspace-based before retirement date(29th February, 2024) to ensure most reliable experience. You can follow steps mentioned in detail [here](https://learn.microsoft.com/en-us/azure/azure-monitor/app/convert-classic-resource#migrate-your-resource) for migration using any one of the below listed approaches.
```

Comment:
migrate **to** classic Application Insights - > migrate classic Application Insights